src/System.Private.CoreLib/shared/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -618,21 +618,21 @@ public static void ThrowExceptionForHR(int errorCode)
         {
             if (errorCode < 0)
             {
-                throw GetExceptionForHR(errorCode, IntPtr.Zero);
+                throw GetExceptionForHR(errorCode, IntPtr.Zero)!;
```

Comment:
The compiler complains for `throw null`... Maybe it shouldn't since that is valid and transforms to `throw NullReferenceException`?

Reply:
> The compiler complains

Sounds like an issue to be opened in the Roslyn repo?

Reply:
Logged https://github.com/dotnet/roslyn/issues/34754